boilerplate/app/models/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from "./RootStore"
-export * from "./helpers/getRootStore"
-export * from "./helpers/useStores"
-export * from "./helpers/setupRootStore"
+export { RootStore, RootStoreModel, RootStoreSnapshot } from "./RootStore"
+export { getRootStore } from "./helpers/getRootStore"
+export { RootStoreProvider, useInitialRootStore, useStores } from "./helpers/useStores"
+export { setupRootStore } from "./helpers/setupRootStore"
```

Comment:
So this almost feels like a blocker for me, to be honest -- I'm very used to being able to export whatever I want and have it be auto-exported from this file.

I think tree-shaking is very interesting, but do we have any stats on how _much_ it's helping in terms of bundle size? This would be a fairly significant DX backwards step (to me).

Reply:
Internally, we're encouraging people to stop using `export *`. It can lead to hard-to-track issues where multiple modules are exporting the same name. Some notes about that here: https://hackmd.io/Z021hgSGStKlYLwsqNMOcg. We also have an ESLint plugin to detect/autofix it: https://github.com/microsoft/rnx-kit/blob/main/packages/eslint-plugin/src/rules/no-export-all.js